### PR TITLE
Merge 2018 19 11 abd4844

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by iMichka** [comment]
Not sure I got the travis part right, this will need some testing/review first.

**Comment by sjackman** [comment]
The TravisCI change would make it test only on macOS on TravisCI, and we probably want to continue testing on both Linux and macOS.

File: .travis.yml [hunk 1]
```diff
@@ -1,51 +1,39 @@
...
-matrix:
-  fast_finish: true
-  include:
-    - os: osx
-      osx_image: xcode10
-    - os: linux
-      sudo: false
+
+env:
+  - HOMEBREW_FORCE_HOMEBREW_ON_LINUX=1
```
Comment (sjackman) [inline .travis.yml]: Let's remove this variable.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -39,6 +39,7 @@ jobs:
       displayName: brew test-bot
       env:
         HOMEBREW_GITHUB_API_TOKEN: $(github.publicApiToken)
+        HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
```
Comment (sjackman) [inline azure-pipelines.yml]: Let's remove this variable.

**Review by sjackman — CHANGES_REQUESTED** [review]

**Pull request merged**